sources/packages/web/src/components/aest/AssessApplicationOfferingChangeRequestModal.vue

```diff
@@ -73,6 +84,7 @@ export default defineComponent({
       resolvePromise(false);
     };
     const assessChange = async () => {
+      processing.value = true;
```

Comment:
actually, for the modal, the proceeding should come from the parent, because actual processing is happening in the parent, you can refer other modals, example, https://vscode.dev/github/bcgov/SIMS/blob/feature/sims-%232289/sources/packages/web/src/components/aest/institution/modals/ApproveProgramModal.vue#L33